code/core/src/manager/components/mobile/navigation/MobileNavigation.tsx

```diff
@@ -134,7 +134,7 @@ const Container = styled.div(({ theme }) => ({
   borderTop: `1px solid ${theme.appBorderColor}`,
 }));
 
-const MobileBottomBar = styled.div({
+const MobileBottomBar = styled.nav({
```

Comment:
(Side note: sites that use multiple `nav`s, like we do, must label each `nav` as they are all landmark regions with keyboard shortcuts to reach them faster; that's why Sidebar is the "Global" `nav`)

I don't think we want to do that because the content is a bunch of buttons that then go on to open the sidebar that contains the main nav.

So here we'd end up with a `nav` that doesn't contain navigation elements, and that opens a `dialog` with another `nav` in it.

We could ask around on the a11y slack if we should include nav above the mobile menu button or under, as I'm not entirely sure what's best, but then we'd want to adjust the sidebar's markup too.

I think in the meantime it's best not to touch this tag.

```suggestion
const MobileBottomBar = styled.div({
```